source/guides/core-concepts/variables-and-aliases.md

```diff
@@ -158,7 +157,7 @@ The reason for using `const` is because the `$span` object is mutable. Whenever
 
 # Aliases
 
-Using `.then()` callback functions to access the previous command values is great - but what happens when you're running code in hooks like `before` or `beforeEach`?
+Using `.then()` callback functions to access the previous command values is great&mdash;but what happens when you're running code in hooks like `before` or `beforeEach`?
```

Comment:
Is there a style guide for the Cypress docs somewhere Jennifer? Even an informal one? I've replaced the ` - ` with an em-dash for style... yes, no?

> I'm not that fussed... the em-dash variant looks better to my eyes but no biggie.

<img width="287" alt="screen shot 2018-08-23 at 22 44 20" src="https://user-images.githubusercontent.com/1855109/44553692-2cfa5800-a726-11e8-97a7-0d457d787aee.png">
<img width="302" alt="screen shot 2018-08-23 at 22 43 41" src="https://user-images.githubusercontent.com/1855109/44553695-31267580-a726-11e8-92be-177c4b944af6.png">

Reply:
👍